packages/react/src/components/Button/index.scss

```diff
@@ -45,77 +73,66 @@
   }
 
   &--primary {
-    background-color: var(--c--theme--colors--primary-500);
+    background-color: var(--c--theme--colors--primary-400);
     color: var(--c--theme--colors--primary-text);
 
-    &:hover {
-      background-color: var(--c--theme--colors--primary-600);
+    &:hover, &:focus-visible {
```

Comment:
I love `focus-visible` but it seems to be really recently added to browsers, don't you think that's is too early? WDYT? 😕

https://caniuse.com/?search=focus-visible